_posts/en/newsletters/2020-09-09-newsletter.md

```diff
@@ -0,0 +1,204 @@
+  which are disadvantaged but serve as a protection against eclipse attacks.
+
+- [Bitcoin Core #14687][] updates the ZMQ event notification service to
+  allow creating connections (sockets) that enable [TCP keepalive][] to
+  help keep connections alive, especially when they pass through a
+  device such as a router or a firewall.  This is especially useful when
+  using zmq to monitor for sometimes-rare events, e.g. for when there's
+  more than 30 minutes between notifications of new blocks.
+
+- [Bitcoin Core #19671][] removes the `-zapwallettxes` configuration
+  option that would delete unconfirmed transactions from the wallet so
+  that users could create and broadcast alternative transactions using
+  the same inputs.  This feature was long ago superseded by the
+  `abandontransaction` RPC (or its equivalent context menu option in the
+  GUI), each of which allow the user to remove specific unconfirmed
+  transactions from the wallet.
+
+- [Bitcoin Core #18244][] updates the `fundrawtransaction` and
+  `walletcreatefundedpsbt` RPCs' `lockunspents` parameter so that it
+  locks user-specified UTXOs (inputs) in addition to those
+  automatically selected by the wallet itself.  Locked UTXOs won't
+  automatically be chosen by the wallet for spending in other
+  transactions until the wallet is restarted or they are manually
+  unlocked; this prevents the user from accidentally creating
+  multiple conflicting transactions that spend the same UTXOs.
+
+- [LND #4463][] adds the ability to create authentication tokens
+  ("macaroons") that only have permission to access a set of APIs
+  specified by the administrator who created the macaroon.  This
+  provides more granular access control than LND's existing
+  category-based permissions system (see [Newsletter #52][news52 lnd
+  acl]).  Additionally, a new `listpermissions` API is added that will
+  list the available APIs and the permissions required to access them.
+
+- [BIPs #983][] updates the [BIP325][] definition of the [signet
+  protocol][topic signet] to document an existing feature of the proposed Bitcoin
+  Core [implementation][bitcoin core #18267] where a challenge that's just
+  `OP_TRUE` (or something else guaranteed to succeed on an empty
+  witness) doesn't need to include a signet commitment in a block's
+  coinbase transaction.
+
+{% comment %}<!-- I'm tempted to link "BitBox 02" to the manufacturer's
+product page, but per a quick `git grep` of the repository, we've never
+linked to the product pages for Trezor, Ledger, or ColdCard, so adding a
+link to for BitBox seems unfair. -harding -->{% endcomment %}
+
+- [HWI #363][] adds support for the BitBox02 hardware wallet.  The
+  initial support only allows signing for single-sig scripts, but
+  multisig support is planned.  According to the PR comments, this is
+  the first merge where HWI support for a new device was contributed on
+  behalf of the manufacturer themselves.
```

Comment:
nit: wordy?
```suggestion
  the first merge where HWI support for a new device was contributed
  from the manufacturer themselves.
```

Reply:
The commits were contributed under the name of a developer rather than the company.  Although this is subtle, I want to recognize that person for their work in addition to the company for their sponsorship.

Reply:
Understood.